packages/expect/src/jest-utils.ts

```diff
@@ -537,6 +537,76 @@ export function generateToBeMessage(deepEqualityName: string, expected = '#{this
   return toBeMessage
 }
 
+const IRREGULAR_PLURALS: { [key: string]: string } = {
+  property: 'properties',
+}
+
 export function pluralize(word: string, count: number): string {
-  return `${count} ${word}${count === 1 ? '' : 's'}`
+  return count === 1 ? word : IRREGULAR_PLURALS[word] ?? `${word}s`
```

Comment:
Can we not change this function?

1. This is a breaking change
2. You can't even use it without adding "irregular" plurals for a single use case where you can just have a single `?` when you call it

Reply:
Alright, agreed. Reverted the change to `pluralize` and just used a ternary.